src/main/java/org/ehcache/sizeof/EhcacheSizeOfEngine.java

```diff
@@ -47,36 +44,26 @@ public Configuration getConfiguration() {
     /**
      * {@inheritDoc}
      */
-    public SizeOfEngine copyWith(int maxDepth, boolean abortWhenMaxDepthExceeded) {
-        return new EhcacheSizeOfEngine(this.cfg);
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    public Size sizeOf(final Object key, final Object value, final Object container) {
+    public Size sizeOf(final Object... objects) {
         Size size;
         try {
-            org.ehcache.sizeof.Size ourSize = sizeOf.deepSizeOf(cfg.getMaxDepth(), cfg.isAbort(), key, value, container);
+            org.ehcache.sizeof.Size ourSize = sizeOf.deepSizeOf(cfg.getMaxDepth(), cfg.isAbort(), objects);
             size = new Size(ourSize.getCalculated(), ourSize.isExact());
         } catch (MaxDepthExceededException e) {
-            logMaxDepthExceededException(key, value, container, e);
+            logMaxDepthExceededException(e);
             size = new Size(e.getMeasuredSize(), false);
         }
 
         if (USE_VERBOSE_DEBUG_LOGGING && LOG.isDebugEnabled()) {
-            LOG.debug("size of {}/{}/{} -> {}", new Object[] { key, value, container, size.getCalculated() });
+            for (int i = 0; i < objects.length; i++) {
+              LOG.debug("size of {} -> {}", new Object[] { objects[i], size.getCalculated() });
+            }
         }
         return size;
     }
 
-    protected void logMaxDepthExceededException(Object key, Object value, Object container, MaxDepthExceededException e)
+    protected void logMaxDepthExceededException(MaxDepthExceededException e)
```

Comment:
This logging got suddenly less verbose... should we be iterating over the var arg and printing properties?